op-node/sources/types.go

```diff
@@ -46,6 +46,7 @@ type HeaderInfo struct {
 	baseFee     *big.Int
 	txHash      common.Hash
 	receiptHash common.Hash
+	gasUsed     uint64
```

Comment:
You need to set this field in `(hdr *rpcHeader) Info`

Reply:
Done